src/SamplesApp/SamplesApp.Skia.netcoremobile/Android/Resources/values/Styles.xml

```diff
@@ -8,5 +8,7 @@
 		<item name="windowNoTitle">true</item>
 		<item name="android:windowNoTitle">true</item>
 
+		<item name="android:windowSplashScreenAnimatedIcon">@drawable/uno</item>
```

Comment:
I am afraid this will not work with the changes Resizetizer is doing 😐 . I will try to look into this too, so please don't merge yet